packages/react-component-library/src/components/Popover/Popover.tsx

```diff
@@ -15,7 +15,7 @@ import { useHideShow } from '../../hooks/useHideShow'
 export interface PopoverProps
   extends Omit<
     FloatingBoxProps,
-    'onMouseEnter' | 'onMouseLeave' | 'renderTarget'
+    'onMouseEnter' | 'onMouseLeave' | 'renderTarget' | 'targetElement'
```

Comment:
A thought here is whether this can be covered by a test. Not a showstopper though.

Reply:
Did you have something specific in mind? It would be a type error to pass `targetElement` (though we have `// @ts-nocheck` on the tests for some reason :man_shrugging: )

Reply:
I was thinking something along the lines of instantiating the component with the omitted props and perhaps wrapping with a `try catch` but I cannot find an easy solution which makes me think it's not worth it.